File: exercises/python/high-scores/high_scores.py
class HighScores(object):
    def __init__(self, scores):
        self.scores = scores

    def latest(self):
        return self.scores[-1]

    def personal_best(self):
        return sorted(self.scores,reverse=True)[0]

    def personal_top_three(self):
        return sorted(self.scores,reverse=True)[:3]

File: exercises/ruby/high-scores/high_scores.rb
class HighScores
  def initialize(scores)
    @scores = scores
  end
  
  def scores
    @scores
  end

  def latest
    @scores.last
  end

  def personal_best
    @scores.sort.reverse.first
  end

  def personal_top_three
    @scores.sort.reverse.first(3)
  end
end

